web/views/api/index_view.ex

```diff
@@ -0,0 +1,8 @@
+defmodule HexWeb.API.IndexView do
+  use HexWeb.Web, :view
+
+  def render("index." <> _format, _assigns) do
+    %{packages_url: api_package_url(Endpoint, :index),
+      documentation_url: "http://docs.hexpm.apiary.io"}
+  end
```

Comment:
oh, there's also `/api/keys` endpoint which we could make discoverable here, but it requires key auth (but prompts for username/password http basic auth when accessed from the browser which won't authenticate)